sparkle/src/components/Tree.tsx

```diff
@@ -161,7 +161,7 @@ Tree.Item = function ({
           <IconButton
             icon={isExpanded ? ArrowDownSIcon : ArrowRightSIcon}
             size="xs"
-            variant="secondary"
+            variant="highlight"
```

Comment:
Why do we sometimes replace `secondary` by ghost and some other times by `highlight`?

Reply:
Yeah I didn't really bother for consistency in the stories but "secondary" should be "highlight". I will make it consistant

Reply:
Should be consistant now